.circleci/config.yml

```diff
@@ -106,13 +111,33 @@ jobs:
               --fork-url="https://ci-sepolia-l1.optimism.io" --compute-units-per-second=320
             forge script CheckSecurityConfigs \
               --fork-url="https://ci-goerli-l1.optimism.io" --compute-units-per-second=320
+      - notify-failures-on-main
 
 workflows:
+  nightly:
+    jobs:
+      - golang-test:
+          context:
+            - slack
+      - check-security-configs:
+          context:
+            - slack
+    triggers:
+      - schedule: 
+          cron: "0 0 * * *"
```

Comment:
Since this job is lightweight, should we run it more frequently to catch issues/changes faster? It shouldn't fail often, but a 24h+ delay between an incorrect config and resolving it feels long. We have a scheduled job in the monorepo that runs hourly, so since there's precedent hourly would be my suggestion, though I don't feel too strongly here

Reply:
Good point, I'm fine with running it hourly.

Reply:
[f0e6afa](https://github.com/ethereum-optimism/superchain-registry/pull/92/commits/f0e6afa9a480067d68aba61bf2d58ec44299b0d9)